--- sympa-3.0-orig/src/Makefile	Thu Dec 14 09:19:32 2000
+++ sympa-3.0/src/Makefile	Fri Jan  5 09:07:03 2001
@@ -17,7 +17,7 @@
 install: 
 	echo "Please use the main Makefile for installing sources."
 
-newinstall: makedir subst $(SRC) installqueue installscenari installtemplates installetc installlisttemplates installmod info
+newinstall: makedir subst $(SRC) installqueue installscenari installtemplates installetc installlisttemplates info
 
 
 installtemplates:
--- sympa-3.0-orig/wwsympa/Makefile	Thu Jan  4 11:37:48 2001
+++ sympa-3.0/wwsympa/Makefile	Fri Jan  5 09:04:07 2001
@@ -10,7 +10,7 @@
 install: 
 	echo "Please use the main Makefile for installing WWSympa."
 
-newinstall: makedir $(SUBSTSRC) substcgi subst installtemplates installicons installsrc installmod
+newinstall: makedir $(SUBSTSRC) substcgi subst installtemplates installicons installsrc
 
 installmod:
 	@for i in $(SRC) wwsympa.fcgi; do \
@@ -81,7 +81,6 @@
 	COL34=70; export COL34; \
 	COL234=110; export COL234; \
 	$(PERL) ../subst.pl wws_templates/* 
-	chmod -R 644 $(DESTDIR)$(ETCBINDIR)/wws_templates
 
 installicons:
 	@echo "Installing icons..."
--- sympa-3.0-orig/src/etc/sample/Makefile	Thu Dec 14 11:15:55 2000
+++ sympa-3.0/src/etc/sample/Makefile	Fri Jan  5 09:04:07 2001
@@ -49,8 +49,6 @@
 		mkdir -p $(DESTDIR)$(DIR)/expl; \
 		mkdir -p $(DESTDIR)$(DIR)/expl/test; \
 		cp test/* $(DESTDIR)$(DIR)/expl/test/; \
-		chown -R $(USER) $(DESTDIR)$(DIR)/expl; \
-		chgrp -R $(GROUP) $(DESTDIR)$(DIR)/expl; \
 	fi
 
 
--- sympa-3.0-orig/Makefile	Thu Jan  4 09:52:05 2001
+++ sympa-3.0/Makefile	Fri Jan  5 09:09:51 2001
@@ -195,8 +195,6 @@
 		if [ ! -d $(DESTDIR)$(DIR)/$$dir ] ; then \
 			echo "Creating $(DESTDIR)$(DIR)/$$dir"; \
 			mkdir -p $(DESTDIR)$(DIR)/$$dir; \
-			chown $(USER) $(DESTDIR)$(DIR)/$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$(DIR)/$$dir; \
 		fi \
 	done \
 	fi
@@ -206,8 +204,6 @@
 		if [ ! -d $(DESTDIR)$(DIR)/$$dir ] ; then \
 			echo "Creating $(DESTDIR)$(DIR)/$$dir"; \
 			mkdir -p $(DESTDIR)$(DIR)/$$dir; \
-			chown $(USER) $(DESTDIR)$(DIR)/$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$(DIR)/$$dir; \
 		fi \
 	done \
 	fi
@@ -218,8 +214,6 @@
 		if [ ! -d $(DESTDIR)$$dir ] ; then \
 			echo "Creating $(DESTDIR)$$dir"; \
 			mkdir -p $(DESTDIR)$$dir; \
-			chown $(USER) $(DESTDIR)$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$$dir; \
 			chmod 770 $(DESTDIR)$$dir; \
 		fi \
 	done \
@@ -247,8 +241,6 @@
 		SSLCERTDIR=$(SSLCERTDIR); export SSLCERTDIR ; \
 		$(PERL) ../../../subst.pl $$cfile \
 	) ;\
-	chown $(USER) $(DESTDIR)$(CONFDIR)/$$cfile; \
-	chgrp $(GROUP) $(DESTDIR)$(CONFDIR)/$$cfile; \
 	fi \
 	done
 
