diff -Nur sympa-3.3.5.org/Makefile.am sympa-3.3.5/Makefile.am
--- sympa-3.3.5.org/Makefile.am	Tue May 14 17:28:21 2002
+++ sympa-3.3.5/Makefile.am	Wed May 15 21:25:47 2002
@@ -292,8 +292,6 @@
 		PIDDIR=$(PIDDIR); export PIDDIR; \
 		$(PERL) ../../subst.pl $$manfile \
 	) ;\
-	chown $(USER) $(DESTDIR)$(MANDIR)/man8/$$manfile; \
-	chgrp $(GROUP) $(DESTDIR)$(MANDIR)/man8/$$manfile; \
 	done
 
 
@@ -310,21 +308,15 @@
 
 installdir:
 	echo "Setting $(USER) owner of $(DESTDIR)$(DIR)"
-	chown $(USER) $(DESTDIR)$(DIR)
-	chgrp $(GROUP) $(DESTDIR)$(DIR)
 	@if [ ! -f $(DESTDIR)$(CONFDIR)/sympa.conf ] ; then \
 	echo "First installation : installing directories..."; \
 	echo "Creating $(DESTDIR)$(DIR)/"; \
 	mkdir -p $(DESTDIR)$(DIR); \
 	chmod 755 $(DESTDIR)$(DIR); \
-	chown $(USER) $(DESTDIR)$(DIR); \
-	chgrp $(GROUP) $(DESTDIR)$(DIR); \
 	for dir in expl etc sample ; do \
 		if [ ! -d $(DESTDIR)$(DIR)/$$dir ] ; then \
 			echo "Creating $(DESTDIR)$(DIR)/$$dir"; \
 			mkdir -p $(DESTDIR)$(DIR)/$$dir; \
-			chown $(USER) $(DESTDIR)$(DIR)/$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$(DIR)/$$dir; \
 		fi \
 	done \
 	fi
@@ -335,8 +327,6 @@
 		if [ ! -d $(DESTDIR)$(DIR)/$$dir ] ; then \
 			echo "Creating $(DESTDIR)$(DIR)/$$dir"; \
 			mkdir -p $(DESTDIR)$(DIR)/$$dir; \
-			chown $(USER) $(DESTDIR)$(DIR)/$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$(DIR)/$$dir; \
 		fi \
 	done \
 	fi
@@ -348,8 +338,6 @@
 		if [ ! -d $(DESTDIR)$$dir ] ; then \
 			echo "Creating $(DESTDIR)$$dir"; \
 			mkdir -p $(DESTDIR)$$dir; \
-			chown $(USER) $(DESTDIR)$$dir; \
-			chgrp $(GROUP) $(DESTDIR)$$dir; \
 			chmod 770 $(DESTDIR)$$dir; \
 		fi \
 	done \
@@ -361,8 +349,6 @@
 	if [ ! -f $(DESTDIR)$(CONFDIR)/$$cfile ] ; then \
 	echo "Installing sample config file $$cfile..."; \
 	$(PERL) -I $(DESTDIR)$(SBINDIR) $(DESTDIR)$(SBINDIR)/sympa_wizard.pl -c $$cfile; \
-	chown $(USER) $(DESTDIR)$(CONFDIR)/$$cfile; \
-	chgrp $(GROUP) $(DESTDIR)$(CONFDIR)/$$cfile; \
 	INITCONF='1'; \
 	fi \
 	done; \
diff -Nur sympa-3.3.5.org/src/Makefile sympa-3.3.5/src/Makefile
--- sympa-3.3.5.org/src/Makefile	Tue May  7 16:18:08 2002
+++ sympa-3.3.5/src/Makefile	Wed May 15 21:31:27 2002
@@ -148,15 +148,6 @@
 	fi
 
 installmod:
-	@echo "Setting group and owner for $(DESTDIR)$(SBINDIR)..."
-	chown -R $(USER) $(DESTDIR)$(SBINDIR)
-	chgrp -R $(GROUP) $(DESTDIR)$(SBINDIR)
-	chown -R $(USER) $(DESTDIR)$(LIBDIR)
-	chgrp -R $(GROUP) $(DESTDIR)$(LIBDIR)
-	chown root $(DESTDIR)$(MAILERPROGDIR)/aliaswrapper
-	chgrp $(GROUP) $(DESTDIR)$(MAILERPROGDIR)/aliaswrapper
-	chown root $(DESTDIR)$(MAILERPROGDIR)/virtualwrapper
-	chgrp $(GROUP) $(DESTDIR)$(MAILERPROGDIR)/virtualwrapper
 	@echo "Setting privileges..."
 	chmod 755 $(DESTDIR)$(SBINDIR) $(DESTDIR)$(ETCBINDIR) 
 	chmod 755 $(DESTDIR)$(ETCBINDIR)/scenari $(DESTDIR)$(ETCBINDIR)/templates $(DESTDIR)$(ETCBINDIR)/list_task_models $(DESTDIR)$(ETCBINDIR)/global_task_models
diff -Nur sympa-3.3.5.org/src/etc/sample/Makefile sympa-3.3.5/src/etc/sample/Makefile
--- sympa-3.3.5.org/src/etc/sample/Makefile	Fri Nov 30 11:39:09 2001
+++ sympa-3.3.5/src/etc/sample/Makefile	Wed May 15 21:27:48 2002
@@ -70,8 +70,6 @@
 		mkdir -p $(DESTDIR)$(DIR)/expl; \
 		mkdir -p $(DESTDIR)$(DIR)/expl/test; \
 		cp test/* $(DESTDIR)$(DIR)/expl/test/; \
-		chown -R $(USER) $(DESTDIR)$(DIR)/expl; \
-		chgrp -R $(GROUP) $(DESTDIR)$(DIR)/expl; \
 	fi
 
 
diff -Nur sympa-3.3.5.org/wwsympa/Makefile sympa-3.3.5/wwsympa/Makefile
--- sympa-3.3.5.org/wwsympa/Makefile	Fri Jan 25 10:34:57 2002
+++ sympa-3.3.5/wwsympa/Makefile	Wed May 15 21:27:48 2002
@@ -38,18 +38,6 @@
 newinstall: makedir $(sbin_SUBST) $(cgi_SUBST) $(lib_SUBST) substcgi substsbin substlib installtemplates installicons installsrc installmod
 
 installmod:
-	@for i in $(SRC) ; do \
-	chown $(USER) $(DESTDIR)$(ETCBINDIR)/$$i; \
-	chgrp $(GROUP) $(DESTDIR)$(ETCBINDIR)/$$i; \
-	done
-	@for i in $(sbin_SUBST) ; do \
-	chown $(USER) $(DESTDIR)$(SBINDIR)/$$i; \
-	chgrp $(GROUP) $(DESTDIR)$(SBINDIR)/$$i; \
-	done
-	@for i in $(cgi_SUBST) ; do \
-	chown $(USER) $(DESTDIR)$(CGIDIR)/$$i; \
-	chgrp $(GROUP) $(DESTDIR)$(CGIDIR)/$$i; \
-	done
 	@echo "Setting SetUID bit on wwsympa.fcgi..."
 	chmod u+s $(DESTDIR)$(CGIDIR)/wwsympa.fcgi
 	chmod g+s $(DESTDIR)$(CGIDIR)/wwsympa.fcgi
@@ -57,8 +45,6 @@
 	chmod 755 $(DESTDIR)$(ICONSDIR)
 	chmod 755 $(DESTDIR)$(LIBDIR)/Marc
 	@for i in $(lib_SUBST) Marc/Search.pm; do \
-	chown $(USER) $(DESTDIR)$(LIBDIR)/$$i; \
-	chgrp $(GROUP) $(DESTDIR)$(LIBDIR)/$$i; \
 	chmod 644 $(DESTDIR)$(LIBDIR)/$$i; \
 	done
 
@@ -124,7 +110,7 @@
 		); \
 	done
 	$(PERL) ../set_symlinks.pl wws_templates $(DESTDIR)$(ETCBINDIR)/wws_templates
-	chmod -R 644 $(DESTDIR)$(ETCBINDIR)/wws_templates
+	chmod -R 644 $(DESTDIR)$(ETCBINDIR)/wws_templates/*
 
 installicons:
 	@echo "Installing icons..."
