diff -urN sympa-3.4.4.3.org/wwsympa/Makefile sympa-3.4.4.3/wwsympa/Makefile
--- sympa-3.4.4.3.org/wwsympa/Makefile	2003-09-04 17:28:22.000000000 +0200
+++ sympa-3.4.4.3/wwsympa/Makefile	2003-09-04 17:40:40.000000000 +0200
@@ -124,7 +124,7 @@
 		); \
 	done
 	$(PERL) ../set_symlinks.pl wws_templates $(DESTDIR)$(ETCBINDIR)/wws_templates
-	chmod -R 644 $(DESTDIR)$(ETCBINDIR)/wws_templates
+	chmod -R 644 $(DESTDIR)$(ETCBINDIR)/wws_templates/*
 
 installicons:
 	@echo "Installing icons..."
