diff -urN sympa-3.4.4.3.org/wwsympa/wwslib.pm sympa-3.4.4.3/wwsympa/wwslib.pm
--- sympa-3.4.4.3.org/wwsympa/wwslib.pm	2003-09-04 17:28:22.000000000 +0200
+++ sympa-3.4.4.3/wwsympa/wwslib.pm	2003-09-04 17:35:23.000000000 +0200
@@ -27,7 +27,7 @@
 @EXPORT = ();
 
 ## Supported web languages
-@languages = ('fr','us','es','it','nl','cn','cz','de','hu','et','ro','fi');
+@languages = ('fr','us','es','it','nl','cn','cz','de','hu','et','ro','fi','pl');
 
 %reception_mode = ('mail' => 'normal',
 		   'digest' => 'digest',
